1906-12-01.xml

```diff
@@ -1484,23 +1468,23 @@
                     LOCAL AND GENERAL.
                 </head>
                     <div type="item">
-                <head>Piagus ot loousts</head>
+                <head>Plague ot loousts</head> <!-- The header was misspelled -->
                 <p>Maize pet ardeb ha» reaohed P.T. 18 at K»*»ala owing ti the recent ravagea
                     ofloenatg.</p></div>
                     <div type="item">
                 <head>Athietlos in the Sudan</head>
-                <p>A now athletio clnb has been started at Khartonm. Tbe inangntati m was attended
-                    by the Sirdar.</p> </div>
+                <p>A new athletio club has been started at Khartonm. Tbe inangntation was attended
+                            by the Sirdar.</p> </div>
                     <div type="item">
-                <head>Bedouin Coneus</head>
-                <p>The cenana of the Bod mins ia fir*t to be taken in the oaae of the tribe*
-                    scattered over the Arabian desert Baat of the Nile.</p> </div>
+                <head>Bedouin Coneus</head> <!-- missing information and very bad grammar -->
```

Comment:
I'd like you to correct the text, rather than describe the problem in a comment--if the corrections are relatively minor